lib/mobility-core/src/Kernel/External/Payment/Interface/Types.hs

```diff
@@ -71,9 +71,36 @@ data CreateOrderReq = CreateOrderReq
     mandateEndDate :: Maybe Text,
     metadataGatewayReferenceId :: Maybe Text,
     optionsGetUpiDeepLinks :: Maybe Bool,
-    metadataExpiryInMins :: Maybe Int
+    metadataExpiryInMins :: Maybe Int,
+    splitSettlementDetails :: Maybe SplitSettlementDetails
   }
 
+data Split = Split
```

Comment:
@0utkarsh All this fields will always be sent by Juspay in SplitSettlementDetails ? Should we make it Maybe if not sure ?